05/app.js

```diff
@@ -1 +1,12 @@
-const numbers = [1, 2, 3, 4, 5, 6, 7];
\ No newline at end of file
+const numbers = [1, 2, 3, 4, 5, 6, 7];
+
+
+const getSumOfEvenNumbers = numbers.filter( isEven ).reduce( function(a, b) {
+    return a + b;
+});
```

Comment:
Tutaj warto pamiętać o drugim parametrze dla `reduce(, 0)`, który określa wartość początkową dla `a` w pierwszej iteracji.
Jeśli go nie ma to pusta tablica spowoduje błąd. Bo nie ma skąd pobrać parametru `a`, który jest pierwszym elementem tablicy, a gdy jest ona pusta to go nie ma.